LOH/1.0.0/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:22.04
+LABEL maintainer="Saksham Phul (phuls@chop.edu)"
+
+WORKDIR home/
+
+RUN apt update && apt upgrade -y && \
+        apt install -y wget && \
+        apt install -y python3.10 && \
+        wget https://github.com/samtools/bcftools/releases/download/1.16/bcftools-1.16.tar.bz2 && \
```

Comment:
you can just list all the apt packages on the same line, no need to call apt one at a time